src/content/add-to-app/ios/project-setup.md

```diff
@@ -86,424 +72,265 @@ my_flutter/
-build configuration. The following instructions assume
-the default **Debug** and **Release**.
-Adjust the names as needed depending on your app's build configurations.
+      {%- render docs/captioned-image.liquid,
+      image:"development/add-to-app/ios/project-setup/set-plist-build-setting.png",
+      caption:"Updating the `Info.plist` build setting to use build
+      mode-specific property lists" %}
 
-<ol>
-<li>
+      This resolves to the path **Info-Debug.plist** in **Debug** and
+      **Info-Release.plist** in **Release**.
 
-Rename your app's **Info.plist** to **Info-Debug.plist**.
-Make a copy of it called **Info-Release.plist** and add it to your Xcode project.
+      {% render docs/captioned-image.liquid,
+      image:"development/add-to-app/ios/project-setup/plist-build-setting.png",
+      caption:"The updated **Info.plist File** build setting displaying the
+      configuration variations" %}
 
-{% render docs/app-figure.md, image:"development/add-to-app/ios/project-setup/info-plists.png", alt:"Info-Debug.plist and Info-Release.plist in Xcode" %}
+1. Remove the **Release** property list from the **Build Phases**.
 
-</li>
+   1. In the **Navigator**, click on your project.
 
-<li>
+   1. Click the **Build Phases** tab.
 
-In **Info-Debug.plist** _only_ add the key `NSBonjourServices`
-and set the value to an array with the string `_dartVmService._tcp`.
-Note Xcode will display this as "Bonjour services".
+   1. Expand **Copy Bundle Resources**.
 
-Optionally, add the key `NSLocalNetworkUsageDescription` set to your
-desired customized permission dialog text.
+   1. If this list includes `Info-Release.plist`,
+      click on it and then click the **-** (minus sign) under it
+      to remove the property list from the resources list.
 
-{% render docs/app-figure.md, image:"development/add-to-app/ios/project-setup/debug-plist.png", alt:"Info-Debug.plist with additional keys" %}
+      {% render docs/captioned-image.liquid,
+      image:"development/add-to-app/ios/project-setup/copy-bundle.png",
+      caption:"The **Copy Bundle** build phase displaying the
+      **Info-Release.plist** setting. Remove this setting." %}
 
-</li>
+1. The first Flutter screen your Debug app loads prompts
+   for local network permission.
 
-<li>
+   Click **OK**.
 
-In your target's build settings, change the **Info.plist File**
-(`INFOPLIST_FILE`) setting path from `path/to/Info.plist` to `path/to/Info-$(CONFIGURATION).plist`.
+   _(Optional)_ To grant permission before the app loads, enable
+   **Settings > Privacy > Local Network > Your App**.
 
-{% render docs/app-figure.md, image:"development/add-to-app/ios/project-setup/set-plist-build-setting.png", alt:"Set INFOPLIST_FILE build setting" %}
+## Mitigate known issue with Apple Silicon Macs
 
-This will resolve to the path **Info-Debug.plist** in **Debug** and
-**Info-Release.plist** in **Release**.
+On an Apple Silicon (M1 or M2) Mac,
```

Comment:
```suggestion
On an [Apple Silicon macs](https://support.apple.com/en-us/116943),
```

There's M3 macs now! There will likely be M4 soon too, as that was recently released for iPad. Perhaps we should just link to Apple's list of Apple Silicon devices instead?